src/inmanta/parser/cache.py

```diff
@@ -27,6 +27,12 @@
 LOGGER = logging.getLogger(__name__)
 
 
+class CacheEnveloppe:
```

Comment:
```suggestion
class CacheEnvelope:
```
Two p's is the Dutch word.

Reply:
done